config/settings.yml

```
@@ -0,0 +1,102 @@
+version: "2024-12-31"
+experiment_run: 1
+
+constants:
+  # Industry names file
+  industry_names: "industry_names.yml"
+  # IO table data year
+  data_year: 2019
+  # First date to simulate
+  first_date: "2020-01-01"
+  # Last date to simulate
+  last_date: "2025-10-01"
+
+flags:
+  stochasticity:
+    # Use data from mat files
+    data_from_mat: false # Replace mat by better format
+
+    # Random initial values for parameters
+    random_initial_values: false
+
+elasticities:
+  consumption:
+    - [3, 2.0]
+  eu_export_demand:
+    - [3, 2.0, 2.0]
+  rest_of_world_export_demand:
+    - [3, 2.0, 2.0]
+  production:
+    - [0.4, 0.75, 2.0]
+  investment:
+    - [0.4, 2.0]
+
+files:
+  input_dir: input
+  output_dir: output
+  scenario: 0 # 0: noCovid, 1: covid
+
+  input_derived_dir: derived
+
+  inputs:
+    base:
+      input_output: "uk_input_output2019.xlsx"
+      imports: "uk_input_output2019.xlsx"
+      merge_codes_64: "mergeCodes64.xlsx"
+      merge_codes_105: "mergeCodes105.csv"
+
+    derived:
+      scenario_dependent:
+        # Time series: income of two types (lo, hi) in two scenarios (0,1) = (noCovid, covid)
+        lo_income: "noCollegeIncome.csv"
+        hi_income: "collegeIncome.csv"
+
+        # Time series: hours worked of two types (lo, hi) in two scenarios (0,1)
+        lo_hours: "noCollegeHours.csv"
+        hi_hours: "collegeHours.csv"
+
+        # Time series: capital in two scenarios (0,1)
+        capital: "capital.csv"
+
+        # Forecast scenarios
+        turnover: "turnover.csv"
+
+      scenario_independent:
+        # Time series: inventories (one scenario only)
+        inventory: "inventories.csv"
+
+        # Base year: depreciation rates
+        depreciation: "depreciation.csv"
+
+        # Financial data
+        RiskFreeRate: "interestRate.csv"
+        assets: "firmData.csv"
+        DefaultSigma: "modelResults.csv"
```

Sorry I thought I made a comment on this yesterday but I can't find it anymore. I'd just like the names of these to follow the standard style. I also don't like `DefaultSigma`. The file seems to contain many variables besides sigma so maybe `model_results` is a reasonable name although it's not very descriptive. We can check with Lars what this is.

```suggestion
        risk_free_rate: "interestRate.csv"
        assets: "firmData.csv"
        model_results: "modelResults.csv"
```

---

Sorry... missed this while renaming